stac_ipyleaflet/core.py

```diff
@@ -382,7 +385,7 @@ def add_tile_layer(
             self.add_layer(tile_layer)
 
         except Exception as e:
-            print("Failed to add the specified TileLayer.")
+            logging.error("Failed to add the specified TileLayer.")
```

Comment:
📓 : Was there a reason why we chose to print errors? It might be better convention to log instead

This is what printing an error looks like in the notebook:
<img width="948" alt="printError" src="https://github.com/MAAP-Project/stac_ipyleaflet/assets/30272083/8ad0efdb-3f6e-4efa-b452-9eb5e0073d8a">

This is what logging an error looks like in the notebook:
<img width="989" alt="loggingError" src="https://github.com/MAAP-Project/stac_ipyleaflet/assets/30272083/686df298-a943-4f4e-80c0-1950420d2b58">